tasties_app/tests/test_category.py

```diff
@@ -0,0 +1,159 @@
+    """
+    This fixture generates some test Recipes
+
+    Returns:
+        tuple<Recipe>: A number of Recipe test objects
+    """
+    user = User.objects.create_user("testuser1", "password")  # This is required to create Recipes
+    recipe_1 = Recipe(
+        title="Test Recipe1",
+        author_id=user,
+        description="Test Description1",
+        directions="Test Directions1",
+        minutes_to_make=1,
+        recipe_picture="test_picture1",
+    )
+    recipe_1.save()
+    recipe_2 = Recipe(
+        title="Test Recipe2",
+        author_id=user,
+        description="Test Description2",
+        directions="Test Directions2",
+        minutes_to_make=1,
+        recipe_picture="test_picture2",
+    )
+    recipe_2.save()
+    return (recipe_1, recipe_2)
+
+
+@pytest.fixture()
+def categories():
+    """
+    This fixture generates some test Categories
+
+    Returns:
+        tuple<Category>: A number of Category test objects
+    """
+    category_1 = Category.objects.create(category_name="Breakfast")
+    category_2 = Category.objects.create(category_name="Mushrooms")
+    category_3 = Category.objects.create(category_name="Beef")
+    return (category_1, category_2, category_3)
+
+
+@pytest.fixture()
+def categorized_recipes(recipes, categories):
+    """
+    This fixture assigns Categories to Recipes
+
+    Args:
+        recipes (fixture): fixture 'recipes'
+        categories (fixture): fixture 'categories'
+
+    Returns:
+        tuple<tuple<Recipe>, tuple<Category>>: The Recipes generated by 'recipes' assigned with
+                       Categories generated by 'categories', as well as the Categories themselves
+    """
+    recipes[0].categories.add(categories[0])
+    recipes[0].categories.add(categories[1])
+    recipes[1].categories.add(categories[1])
+    return recipes, categories
+
+
+class TestCategoryModel:
+    @pytest.mark.django_db
+    def test_create_categories(self, categories, recipes):
+        """
+        Verifies fixtures 'categories' and 'recipes' properly
+           saved their data as expected
+
+        Args:
+            categories (fixture): fixture 'categories'
+            recipes (fixture): fixture 'recipes'
+        """
+        all_categories = Category.objects.all()
+        all_recipes = Recipe.objects.all()
+        for category in categories:
+            assert category in all_categories
+        for recipe in recipes:
+            assert recipe in all_recipes
+
+    @pytest.mark.django_db
+    def test_get_recipes_by_category(self, categorized_recipes):
+        """
+        Verifies method 'get_recipes_by_category' provides intended functionality
+
+        Args:
+            categories (fixture): fixture 'categories'
+            recipes (fixture): fixture 'recipes'
+            categorized_recipes (fixture): fixture 'categorized_recipes'
+        """
+        # define test variables
+        recipes = categorized_recipes[0]
+        categories = categorized_recipes[1]
+
```

Comment:
I think this blank line is redundant.

Reply:
> I think this blank line is redundant.

Me too. However, flake8 disagrees with us unfortunately :(

Reply:
I think it contributes to readability - i.e. there's a visual partition that makes it clear that we've extracted the tuples `recipes` and `categories` from the tuple-of-tuples `categorized_recipes` gave us (it stands out a bit). Regardless, I don't think it will make this bit of code easier to CR if I remove the space - only possibly more difficult, in my opinion.

Reply:
Thanks, let's keep it as is